app/src/app.py

```diff
@@ -34,6 +40,29 @@ def create_app() -> APIFlask:
     return app
 
 
+def is_running_in_local() -> bool:
+    print('s.getenv("FLASK_ENV") => ', os.getenv("FLASK_ENV"))
+    return os.getenv("FLASK_ENV") in ("local", "dev")
```

Comment:
The environment is just `ENVIRONMENT`. I'd also restrict it to only `local` as some projects use `dev` as a non-local environment.